roles/sysbench/tasks/main.yml

```diff
@@ -0,0 +1,42 @@
+---
+# tasks file for sysbench
+- name: template sysbench script
+  k8s:
+    definition:
+      apiVersion: v1
+      kind: ConfigMap
+      metadata:
+        name: '{{ meta.name }}-sysbench-config'
+        namespace: '{{ meta.namespace }}'
+      data:
+        sysbenchScript: "{{ lookup('template', 'sysbench.sh.j2') }}"
+
+- name: start sysbench job
+  k8s:
+    definition:
+      kind: Job
```

Comment:
You could just use a pod here instead of a job? Any specific reason for the Job?

Reply:
I preferred the job because it terminated the pod as well. Just felt that was cleaner.

Reply:
Terminates pods but holds onto logs.